{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2022-43855",
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "state": "PUBLISHED",
        "assignerShortName": "ibm",
        "dateReserved": "2022-10-26T15:46:22.823Z",
        "datePublished": "2024-03-08T17:52:57.326Z",
        "dateUpdated": "2024-07-04T11:59:31.630Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unaffected",
                    "product": "SPSS Statistics",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "26.0, 27.0.1, 28.0"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "IBM SPSS Statistics 26.0, 27.0.1, and 28.0 could allow a local user to create multiple files that could exhaust the file handles capacity and cause a denial of service.  IBM X-Force ID:  230235."
                        }
                    ],
                    "value": "IBM SPSS Statistics 26.0, 27.0.1, and 28.0 could allow a local user to create multiple files that could exhaust the file handles capacity and cause a denial of service.  IBM X-Force ID:  230235."
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "attackComplexity": "LOW",
                        "attackVector": "LOCAL",
                        "availabilityImpact": "HIGH",
                        "baseScore": 6.2,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "NONE",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
                        "version": "3.1"
                    },
                    "format": "CVSS",
                    "scenarios": [
                        {
                            "lang": "en",
                            "value": "GENERAL"
                        }
                    ]
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-399",
                            "description": "CWE-399 Resource Management Errors",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm",
                "dateUpdated": "2024-03-08T17:52:57.326Z"
            },
            "references": [
                {
                    "tags": [
                        "vendor-advisory"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/7130881"
                },
                {
                    "tags": [
                        "vdb-entry"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/239235"
                }
            ],
            "source": {
                "discovery": "UNKNOWN"
            },
            "title": "IBM SPSS Statistics denial of service",
            "x_generator": {
                "engine": "Vulnogram 0.1.0-dev"
            }
        },
        "adp": [
            {
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "id": "CVE-2022-43855",
                                "role": "CISA Coordinator",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "version": "2.0.3",
                                "timestamp": "2024-03-11T14:09:22.860645Z"
                            }
                        }
                    }
                ],
                "providerMetadata": {
                    "orgId": "8c464350-323a-4346-a867-fc54517fa145",
                    "shortName": "CISAADP",
                    "dateUpdated": "2024-05-24T01:06:22.252Z"
                }
            },
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-07-04T11:59:31.630Z"
                },
                "references": [
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/7130881/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/239235/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/7130881/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/239235/adp-test-6-28"
                    }
                ]
            }
        ]
    }
}